oereb_bundesgesetze/build.gradle

```diff
@@ -16,9 +16,10 @@ def pathToUnzipFolder = Paths.get(pathToTempFolder, "unzip_data")
 def pathToDataZip = Paths.get(pathToTempFolder, "data.zip")
 
 // Oereb-DB Schemas
-def dbSchemas = []
-dbSchemas.add(["stage"])
-dbSchemas.add(["live"])
+def dbSchemas = [
```

Comment:
Hm, schreib' doch alles auf eine Zeile, so wie z.B. in https://github.com/sogis/oereb-gretljobs/blob/92bd4260406c9a1f511e3ff5e00bdff70ee064cb/oereb_av/build.gradle#L9 Weil die Anzahl Elemente des Arrays ja sehr überschaubar ist.